AUTHORS

```diff
@@ -0,0 +1,27 @@
+# Directly affiliated with DEDIS
+Bryan Ford <bryan.ford@epfl.ch>
+Kokoris Kogias Eleftherios <eleftherios.kokoriskogias@epfl.ch>
+Ismail Khoffi <ismail.khoffi@epfl.ch>
+Linus Gasser <linus.gasser@epfl.ch>
+Nicolas Gailly <nicolas.gailly@epfl.ch>
+Nikitin Kirill <kirill.nikitin@epfl.ch>
+Philipp Jovanovic <philipp.Jovanovic@epfl.ch>
+
+# NON-DEDIS - probably need a CAA or CLA
```

Comment:
Maybe this was because they aren't directly affiliated with the DEDIS lab (which seems to be the copyright holder). Also, I wonder, wasn't their code "moved"? But I guess this doesn't change anything.